app/models/manageiq/providers/redhat/infra_manager/inventory/strategies/v4.rb

```diff
@@ -8,7 +8,7 @@ def initialize(args)
     end
 
     def host_targeted_refresh(target)
-      ems.with_provider_connection(:version => 4) do |connection|
+      @ems.with_provider_connection(:version => 4) do |connection|
```

Comment:
Since this is in the `v4.rb` file would it make sense to make either the `4` or the whole hash a constant.

```
VERSION = 4
VERSION_HASH = {:version => VERSION}.freeze
...
@ems.with_provider_connection(VERSION_HASH) do |connection|
```